source/Sylvan.Data.Csv/Exceptions.cs

```diff
@@ -7,8 +7,8 @@ namespace Sylvan.Data.Csv;
 /// </summary>
 public class CsvFormatException : FormatException
 {
-	internal CsvFormatException(int row, int ordinal, string? msg = null, Exception? inner = null)
-		: base(msg, inner)
+	internal CsvFormatException(int row, int ordinal, string message, Exception? inner = null)
```

Comment:
Made `message` mandatory (non-nullable), so that any future uses of this exception would require a message to be passed in as well.